jenkins_cli/cli.py

```diff
@@ -31,6 +32,11 @@
                   }
 
 
+# Default constants to parse booleans from .jenkins-cli
+TRUE = ['yes', 'Yes', 'YES', 'True', 'true', 'TRUE', '1']
```

Comment:
I think if ```ignore_ssl=``` option is in ```.jencins-cli``` file, that means that we do want to ignore ssl errors. So I think we do not need ```TRUE``` constant as well.